roles/neutron-data-network/tasks/ipchanged.yml

```diff
@@ -19,6 +19,7 @@
 - name: configure neutron external services failover
   template: src=etc/ipchanged/add_floating_ip mode=0755
             dest=/etc/ipchanged/{{ ansible_default_ipv4.interface }}/{{ floating_ip }}/add
+  when: neutron.neutron_as_gateway.enabled|default('False')|bool
```

Comment:
We need to change `neutron_as_gateway` option name so it accurately reflects the configuration this option is used in